src/render.js

```diff
@@ -147,26 +149,27 @@ const render = async function(
         if (globals) {
             Object.keys(globals).forEach(key => {
                 // Location is a special case.
-                if (key === 'location') {
-                    context.location.replace(globals[key]);
-                } else {
-                    context[key] = globals[key];
+                if (key !== 'location') {
+                    context.window[key] = globals[key];
                 }
             });
         }
 
         // If Apollo is required, get it configured on the context.
-        if (context.ApolloNetwork) {
-            configureApolloNetwork(context);
+        if (context.window.ApolloNetwork) {
```

Comment:
Nice, this better than copying everything to the `context`.

Reply:
Copying the the context would no longer work now. Since JSDOM properly sets up the vm, it sets window as the root VM object instead of the context, so if we hung things off context, our code wouldn't be able to see it. Really, our old version of this code was doing the wrong thing because it was making context be a window, when really, it shouldn've just been building the vm off of window like JSDOM now does.